scipy/interpolate/_cubic.py

```diff
@@ -1,7 +1,5 @@
 """Interpolation algorithms using piecewise cubic polynomials."""
 
-from __future__ import annotations
```

Comment:
be careful with the remaining `TYPE_CHECKING`

Reply:
removed